sheet_to_triples/run.py

```diff
@@ -14,19 +14,32 @@
 
 default_model = object()
 
+# TODO: this should not hardcode RDF namespaces
+_default_non_unique = {'http://visual-meaning.com/rdf/classOfInterest'}
```

Comment:
Should also probably not live here? The runner shouldn't care about ontology specifics.

Reply:
Only solution I can think of is to kick it upstairs to `main`, which also feels wrong. Will land as-is and revisit later.